Header header

Payload[] payloads
PayloadTarget[] payload_targets
LinkMarkers[] link_markers
#Payload, PayloadTarget, and LinkMarkers names to remove
string[] delete_payloads
